gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,8 +66,23 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
```

Comment:
This method doesn't seem necessary. Can't we just call `queue_failover_executed_event` directly?